runtime/exelib/common/rconsole.c

```diff
@@ -60,7 +60,7 @@ remoteConsole_parseCmdLine(J9PortLibrary *portLibrary, UDATA lastLegalArg, char
 	port = NULL;
 	filepath = NULL;
 	for (i = 1; i <= lastLegalArg; i++) {
-		if ((argv[i][0] == '-')) {
+		if (argv[i][0] == '-') {
```

Comment:
Can you swap the arguments so the constant is on the left?

Reply:
did you want me to go through the rest of the function / file or just submit the swap for this one line for now ?

Reply:
Actually, I think that sort of 'cleanup' should be done in a separate PR.
You will need to sign the Eclipse Contributor Agreement (ECA).

Reply:
I have a ECA signed yesterday so hopefully all in the system by now

Reply:
It is weird.  You did not fail the IP-Validation.  It says that the service is down.

Reply:
The format of that line (in your commits) should be:
```
Signed-off-by: Steve Wallin <steve_wallin@uk.ibm.com>
```